### ISSUE TITLE
how to set arbitrary padding value to conv2d filter?

### ISSUE THREAD

**Issue by eric2323223** [keras-team/keras#9508]
I am trying to implement inception module myself and i find both 'valid' and 'same' padding cannot meet my need. for instance, i would like to get 9×9 feature map from applying a 5×5 filter to 11×11 input, could anyone tell me how to do it?

**Comment by Dref360** [comment]
https://keras.io/layers/convolutional/#zeropadding2d

**Comment by thefiddler** [comment]
Use ZeroPadding2D().
